src/cogs/jaegeraccounts.py

```diff
@@ -294,31 +299,52 @@ async def distribute_accounts(self, ctx: commands.Context, force="False"):
         async with dbPool.acquire() as conn:
             url = await conn.fetchval("SELECT url FROM sheet_urls WHERE fk = (SELECT id FROM guilds WHERE guild_id = $1);", ctx.guild.id)
         if url is None:
-            raise NoSheetsUrlException("There is no google sheets url associated with this guild.")
+            raise NoSheetsUrlException(
+                "There is no google sheets url associated with this guild.")
```

Comment:
Looks like autoformatting, either on my old code or on your end.
```suggestion
            raise NoSheetsUrlException("There is no google sheets url associated with this guild.")
```